### PR TITLE
fix: update source config responses

### PR DESCRIPTION
**Pull request by saikumarrs**
## PR Description

I've updated the expected source config responses according to the upcoming trimmed responses from the API endpoint.

## Linear task (optional)

https://linear.app/rudderstack/issue/SDK-1893/update-sanity-suite-as-per-the-changes-to-source-config-response

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [x] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [x] All sanity suite test cases pass locally

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

## Summary by CodeRabbit

- **Improvements**
  - Updated column header in tables from "Expected Payload" to "Generated Payload" for better clarity.
  - Improved accuracy in mutation observer by adjusting the index for accessing child nodes.

- **Removed Features**
  - Removed properties related to live events configuration and destinations from the source configuration to streamline the system.

### CODE REVIEW

File: packages/sanity-suite/__fixtures__/sourceConfig1.json [hunk 1]
```diff
@@ -28,19 +29,11 @@
...
         },
-        "liveEventsConfig": {},
         "destinationDefinitionId": "1X6kXapk8Sf8yZTJn1dYZO3OXFt",
         "destinationDefinition": {
           "name": "GTM",
           "displayName": "Google Tag Manager"
         },
-        "deleted": false,
-        "createdAt": "2023-02-21T05:58:04.724Z",
-        "updatedAt": "2023-11-06T10:42:57.261Z",
```
Comment (MoumitaM) [inline packages/sanity-suite/__fixtures__/sourceConfig1.json:39]: updatedAt has not been removed

File: packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js [hunk 2]
```diff
@@ -7,178 +7,6 @@ const sourceConfigIgnoredProperties = [
...
-    type: 'string',
-  },
-  {
-    key: `source.destinations.0.liveEventsConfig.eventDeliveryTS`,
-    type: 'number',
-  },
-  {
-    key: `source.destinations.0.updatedAt`,
-    type: 'string',
-  },
```
Comment (MoumitaM) [inline packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js:25]: We need to keep this

Comment (MoumitaM) [reply packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js:25]: Same goes for other destinations

File: packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js [hunk 3]
```diff
@@ -7,178 +7,6 @@ const sourceConfigIgnoredProperties = [
...
-  {
-    key: `source.destinations.9.destinationDefinition.updatedAt`,
-    type: 'string',
-  },
-  {
-    key: `source.updatedAt`,
-    type: 'string',
-  },
```
Comment (MoumitaM) [inline packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js:181]: This is also present

**Review by MoumitaM — CHANGES_REQUESTED** [review]
updatedAt field under each destination is not removed

File: packages/sanity-suite/__fixtures__/sourceConfig1.json (hunk 1 above)
Comment (saikumarrs) [reply packages/sanity-suite/__fixtures__/sourceConfig1.json:39]: Addressed.

File: packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js (hunk 2 above)
Comment (saikumarrs) [reply packages/sanity-suite/src/ignoredProperties/sourceConfigIgnoredProperties.js]: Addressed.

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**